2-mandatory/2-financial-times.js

```diff
@@ -5,7 +5,17 @@
     Implement the function below, which will return a new array containing only article titles which will fit.
 */
 function potentialHeadlines(allArticleTitles) {
-    // TODO
+  // TODO
+
+  let lessArticle = [];
+
+  for (headline of allArticleTitles) {
```

Comment:
The code looks good here.
Same comment as above about using `let` or `const` for the `headline` variable here.